### ISSUE TITLE
Document using react-router v4 with kasia and universal rendering

### ISSUE THREAD

**Issue by nickpalmer** [outlandishideas/kasia#63]
I got this working today and it might be nice to tell other people how to do it.

**Comment by joshlevinson** [comment]
Would be very excited to see this–this was my first thought after discovering this (great!) library and spending a few minutes reading the readme 😄

**Comment by sdgluck** [comment]
I haven't used v4 of react-router so couldn't do this without some serious digging (the API is vastly different at first glance). Happy to accept PRs though! 😸

**Comment by josephmartz** [comment]
Same here. The API is totally different but this library works fluently <3

**Comment by AnalogMemory** [comment]
@nickpalmer Do you have a sample of how you got it running?

**Comment by nickpalmer** [comment]
Not that is separate at this point, and I don't think I will be able to break it out into a demo project for at least a month, if not more.

**Comment by MickeyKay** [comment]
Are folks still interested in this? If so, @joshlevinson and I have an example we could formalize. It's a bit specific to our org's share infrastructure, but I imagine might still be useful.

**Comment by sdgluck** [comment]
@MickeyKay Yes we certainly would be interested in seeing an example, if you have one to share. Has it been part of your work re: #80?
